src/components/asthma/components/AsthmaActionPlanManager/AsthmaActionPlanManager.tsx

```diff
@@ -0,0 +1,102 @@
+import React, { useState } from 'react';
+import './AsthmaActionPlanManager.css';
+import { Button, LoadingIndicator, UnstyledButton } from '../../../presentational';
+import MyDataHelps from '@careevolution/mydatahelps-js';
+import { useInitializeView } from '../../../../helpers/Initialization';
+import sampleActionPlan from '../../assets/sample_aap.png'
```

Comment:
I would not bundle this with the app but instead host this somewhere else (e.g. just upload it with MDHD).  Otherwise it will add 50kb to the view builder package for instance...which seems unnecessary

Reply:
Done.